curriculum/challenges/english/blocks/lecture-understanding-form-validation/6733d3a33abdd27cd562bdf2.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 6733d3a33abdd27cd562bdf2
-title: What Is the Purpose of e.preventDefault?
+title: What is the Purpose of the preventDefault() Method?
 challengeType: 19
-dashedName: what-is-the-purpose-of-e-preventdefault
+dashedName: what-is-the-purpose-of-the-preventdefault-method
```

Comment:
was it confirmed in the issue that we want to also update the dashedName?

this requires creating redirects